app/controllers/search_controller.rb

```diff
@@ -0,0 +1,13 @@
+class SearchController < ApplicationController
+  def search
+    if params[:q]
+      if params[:q] == ''
+        @cities = []
+        redirect_to '/search', notice: "You must input a character"
+      else
+        @cities = City.where('name ILIKE ?', "%#{params[:q]}%")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.